src/StmtToViz.cpp

```diff
@@ -740,7 +734,7 @@ class HTMLCodePrinter : public IRVisitor {
             print_html_element("span", "matched", " {");
 
             // Open indented div to hold buffer data
-            print_opening_tag("div", "indent BufferData", id);
+            print_opening_tag("div", "indent BufferData collapsed-block", id);
```

Comment:
Question: why are we printing `buffer cuda_gpu_source_kernels` in the StmtVisualizer's Stmt panel? I suppose I can collapse the assembly code by default.

Reply:
In this instance, the stmt tab is just mimicking the basic IR printer behavior.

Given that we now have an assembly tab when using the html output, perhaps we can instead simply add a jump to the corresponding line in assembly. I think collapsing for now is fine as well.

Reply:
Re: collapsing the block. Great. 

Re: button to cross-reference IR with assembly. As far as I know, the feature is absent in Halide 16.0.0. Line 2164 appears to destroy the bookmarks on the fly. Not sure why.

I don't use that feature by the way. Wondering who else is using it?
 https://github.com/halide/Halide/blob/027547f713f6a01c7b51cb59baa665219f89751d/src/StmtToViz.cpp#L2164

Reply:
> Question: why are we printing buffer cuda_gpu_source_kernels in the StmtVisualizer's Stmt panel? I suppose I can collapse the assembly code by default.

It boils down to the other comment of @maaz139. Because that really _is_ the lowered IR. See: https://github.com/halide/Halide/pull/7753/files#r1290639013

I think it would be wrong to take it out.